### PR TITLE
LPS-186596 and LPS-185983 Adds Portlet to render the view and a resource command to generate text using openAI API

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-186596
https://issues.liferay.com/browse/LPS-185983

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by ealonso** [comment]
Hey @lfbesada,
main.css file is not necessary

File: modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/portlet/AICreatorOpenAIPortlet.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		"javax.portlet.name=" + AICreatorOpenAIPortletKeys.AI_CREATOR_OPENAI,
+		"javax.portlet.resource-bundle=content.Language",
+		"javax.portlet.version=3.0"
+	},
+	service = Portlet.class
+)
+public class AICreatorOpenAIPortlet extends MVCPortlet {
+
+	@Override
+	public void render(
```
Comment (ealonso) [inline modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/portlet/AICreatorOpenAIPortlet.java:55]: @lfbesada Why we need a portlet? Is it adding some benefits? Why not using a struts action?

**Comment by lfbesada** [comment]
@ealonso Thanks a lot for your feedback ❤️ I will resend this pull request taking into account your comments. Regards

**Pull request closed without merging**